client/web/web.go

```diff
@@ -609,6 +613,14 @@ func (s *Server) serveGetNodeData(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 	filterRules, _ := s.lc.DebugPacketFilterRules(r.Context())
+
+	pathPrefix := strings.TrimSuffix(s.pathPrefix, "/")
+	if hostinfo.GetEnvType() == hostinfo.HomeAssistantAddOn && pathPrefix == "" {
```

Comment:
nit: Could move this below the initial node data construction? Then that way the default can just be set down there.

Reply:
oh, good point.